week05/141. Linked List Cycle.cpp

```diff
@@ -0,0 +1,23 @@
+//141. Linked List Cycle
+//Time Complexity : O(N)
+//Space Complexity : O(1)
+
+class Solution {
+public:
+    bool hasCycle(ListNode *head) {
+        if(head == NULL) return false;
+        auto tortoise = head;
+        auto rabbit = head;
+        while(rabbit && rabbit->next)
+        {
+            tortoise = tortoise->next;
+            rabbit = rabbit->next->next;
+            if(tortoise == rabbit)
```

Comment:
DO NOT name your variables like `rabbit` and `tortoise`. They DO NOT go with the context here. Let's say, I don't know what is rabbit and tortoise at all (I am a non-english speaking person). Then how do I understand this?

Use `slow` and `fast` instead.